torch_xla/csrc/tensor_methods.cpp

```diff
@@ -962,9 +946,16 @@ XLATensor XLATensor::clamp(const XLATensor& input,
 XLATensor XLATensor::clamp(const XLATensor& input,
                            const c10::optional<at::Tensor>& min,
                            const c10::optional<at::Tensor>& max) {
-  MinMaxValues min_max = GetMinMaxValues(input, min, max);
-  return input.CreateFrom(
-      ir::ops::Clamp(input.GetIrValue(), min_max.min, min_max.max));
+  XLA_CHECK(min || max)
+      << "At least one of \'min\' or \'max\' must not be None";
+  ir::Value res = input.GetIrValue();
+  if (min) {
+    res = ir::ops::Max(res, bridge::GetXlaTensor(*min).GetIrValue());
```

Comment:
Do we still use `ir::ops::Clamp` elsewhere? ;)

Reply:
yes, there is a clamp op that takes `scalar`, we can still use `ir::ops::Clamp` for that.